### DISCUSSION TITLE
Can't call setState on a component that is not yet mounted Warning occurs when use react-draft-wysiwyg with react-hook-team

### DISCUSSION THREAD

**Discussion by Xiang-Zuo** [react-hook-form/react-hook-form#4750] [Q&A]
Hello team, 

I am currently writing a project using react-hook-form to render forms that contain the rich-text fields. The library I used to perform the rich-text field is react-draft-wysiwyg. For the first click on the text editor area after refreshing the page, I saw the warning printed on the console says ```Can't call setState on a component that is not yet mounted```. I searched some sample codes on CodeSandBox that use react-hook-form & react-draft-wysiwyg and most of them have similar warnings printed on the console. [One of the examples.](https://codesandbox.io/s/wizardly-williams-2cjoe)

I am curious if this is a common problem? Is it because the RichTextEditor is wrapped inside the Controller? And is there any possible solution to solve this problem? Please kindly advise, thank you! 
I am using V7 react-hook-form with react-draft-wysiwyg version 1.14.5. 

### Below is my code:

```js
<Controller
    render={({field: {value, onChange}}) => {
        return (
            <>
                <Editor
                    editorState={value}
                    onEditorStateChange={editorState => {
                        onChange(editorState)
                    }}
                />
            </>
        );
    }}
    name={fieldName}
    control={control}
    defaultValue={fieldValue ? EditorState.createWithContent(getContentFromHTML(fieldValue)) : EditorState.createEmpty()} // field value is stored as string initially
/>
```
### Warning message:
![Screen Shot 2021-04-12 at 8 35 58 PM](https://user-images.githubusercontent.com/35638284/114479528-b9573300-9bce-11eb-9321-46b8799199eb.png)

**Comment by bluebill1049** [comment]
<img width="739" alt="Screen Shot 2021-04-22 at 12 58 01 pm" src="https://user-images.githubusercontent.com/10513364/115649276-60279600-a36a-11eb-83de-de174d57fe68.png">

I think you are using v6, but strange problem tho. Maybe you should use just pure `register` and `setValue` API instead of `Controller`.
